accounts/doc.go

```diff
@@ -18,5 +18,19 @@
 // This file is derived from accounts/accounts.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package accounts implements high level Klaytn account management.
+/*
+Package accounts implements high level Klaytn account management.
+
+There are several important data structures and their hierarchy is as below
+ Manager -- Backend -- Wallet -- Account
+
+Source Files
+
+Each file provides the following features
+ - accounts.go	: Provides `Account` struct and Wallet/Backend interfaces. `Account` represents a Klaytn account located at a specific location defined by the optional URL field. `Wallet` represents a software or hardware wallet and `Backend` is a "wallet provider" that may  contain a batch of accounts they can sign transactions with
```

Comment:
Just curious, what is difference between `blah` and "blah"?

Reply:
@KimKyungup I think the double quotation marks of "wallet provider" should be removed. I used `Something` for structs and interfaces. " " was just for attention.